.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -3,7 +3,7 @@ name: Feature request
 about: Suggest an idea for this project
 title: "[Feature] "
 labels: kind/feature
-assignees: thetechnick
+assignees: ''
```

Comment:
why not remove the key as well?

Reply:
just wanted to keep it the same as the bug_report.md

Reply:
I wondered already why so much is getting assigned to me :D